app/case/confirm_completion.mustache

```diff
@@ -0,0 +1,29 @@
+<div>
+
+    <p>
+        Please confirm that the following date is the completion date to be added to the mortgage deed.
+    </p>
+
+    <p>
+        <b>{{current_date}}</b>
+    </p>
+
+    <p>
+        Once you have confirmed this date, you will not be able to edit the mortgage deed.
+    </p>
+
+    <p>
+        <form method="post">
+            <input type="submit"
+                   name="confirm-completion"
+                   id="confirm-completion"
+                   value="Confirm completion"
+                   class="button"/>
+        </form>
+    </p>
+
+    <p>
+        <a href="{{case_list_url}}">Cancel</a>
+    </p>
+
+</div>
```

Comment:
just a not pick but its complaining there is no new line at the end of the file here